Sources/protoc-gen-swift/GeneratorOptions.swift

```diff
@@ -15,11 +15,35 @@ class GeneratorOptions {
     case fullPath = "FullPath"
     case pathToUnderscores = "PathToUnderscores"
     case dropPath = "DropPath"
+
+    init?(rawValue: String) {
```

Comment:
I would give the argument label here and below a different name, like `init?(flag: String)`, since it's no longer a 1:1 mapping between the actual raw values, accepting that this will also require updating the call-sites.

Going further, do we use the raw values for anything else other than parsing the command line flag? If not, we should just drop them and remove the `String` type from the enum.

Reply:
Done.